### DISCUSSION TITLE
dns plugin attr in strongswan.conf will overwrite the left|rightdns setting in ipse.conf?

### DISCUSSION THREAD

**Discussion by TomHsiung** [strongswan/strongswan#1623] [General]
If the rightdns parameter in `/etc/ipsec.conf` is used to assign dns server for client (roadwarrior mode), will they dispute with settings in `/etc/storngswan.conf` by plugins attr?

Thanks.

**Comment by tobiasbrunner** [comment]
The _attr_ plugin always adds all configured attributes to the response. You might want to use swanctl.conf instead, where you can [define attributes per address pool](https://docs.strongswan.org/docs/5.9/swanctl/swanctlConf.html#_pools).
